addons/binding/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/handler/ApplianceStatusListener.java

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
That's an interesting header. Best if you could re-run `mvn license:format` over the whole codebase.